python/fbprophet/forecaster.py

```diff
@@ -198,21 +198,30 @@ def validate_column_name(self, name, check_holidays=True,
         reserved_names.extend([
             'ds', 'y', 'cap', 'floor', 'y_scaled', 'cap_scaled'])
         if name in reserved_names:
-            raise ValueError('Name "{}" is reserved.'.format(name))
+            raise ValueError(
+                'Name "{name}" is reserved.'.format(name=name)
+            )
         if (check_holidays and self.holidays is not None and
                 name in self.holidays['holiday'].unique()):
             raise ValueError(
-                'Name "{}" already used for a holiday.'.format(name))
+                'Name "{name}" already used for a holiday.'.format(name=name)
+            )
         if (check_holidays and self.country_holidays is not None and
                 name in get_holiday_names(self.country_holidays)):
             raise ValueError(
-                'Name "{}" is a holiday name in {}.'.format(name, self.country_holidays))
+                'Name "{name}" is a holiday name in {country_holidays}.'
+                .format(name=name, country_holidays=self.country_holidays)
+            )
         if check_seasonalities and name in self.seasonalities:
             raise ValueError(
-                'Name "{}" already used for a seasonality.'.format(name))
+                'Name "{name}" already used for a seasonality.'
+                .format(name=name)
+            )
```

Comment:
See above comment.

Reply:
Sorry it took me so long. Addressed these in [PR 1301](https://github.com/facebook/prophet/pull/1301)